template.php

```diff
@@ -148,11 +148,17 @@ function campaignion_foundation_form_alter(&$form, $form_state, $form_id) {
   if (empty($form['actions']['#type']) || $form['actions']['#type'] !== 'actions') {
     return;
   }
-  // Add submit button classes.
-  $button_classes = ['small-only-expanded', 'primary', 'button'];
-  foreach (array_values($button_classes) as $class) {
-    $form['actions']['next']['#attributes']['class'][] = $class;
-    $form['actions']['submit']['#attributes']['class'][] = $class;
+  // Edit submit button classes.
+  foreach (array_values(['next', 'submit']) as $type) {
+    $button_classes = &$form['actions'][$type]['#attributes']['class'];
+    // Add submit button classes.
+    $classes = ['small-only-expanded', 'primary', 'button'];
```

Comment:
I don't see any advantage in setting the variable in each array loop? :thinking:

Reply:
yes :)